src/auth/token.service.ts

```diff
@@ -26,8 +28,12 @@ export class TokenService {
       secret: process.env.REFRESH_TOKEN_SECRET,
     });
   }
-  // async setRefreshToken(userId: string, refreshToken: string): Promise<void> {
-  //   const ttl = this.configService.get("JWT_REFRESH_TOKEN_EXPIRATION_TIME"); // TTL 값 설정(리프레시 토큰 만료 시간을 TTL로 지정 만료 이후엔 캐시에서 토큰 삭제)
-  //   await this.cacheService.set(`refreshToken:${userId}`, refreshToken, +ttl);
-  // }
+
+  async setRefreshToken(userNo: string, refreshToken: string, ttl: number) {
+    return this.redisService.setToken(userNo, refreshToken, ttl);
+  }
+
+  async setAccessToken(userNo: string, accessToken: string, ttl: number) {
```

Comment:
```suggestion
  setAccessToken(userNo: string, accessToken: string, ttl: number) {
```
여기도 async 없어도 될거 같아용